packages/asset/contracts/LazyVault.sol

```diff
@@ -0,0 +1,131 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.8.18;
+
+import {AccessControl} from "@openzeppelin/contracts/access/AccessControl.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {ILazyVault} from "./interfaces/ILazyVault.sol";
+
+/// @title LazyVault
+/// @author The Sandbox
+/// @notice Contract responsible for distributing tokens to creators and split recipients
+contract LazyVault is ILazyVault, AccessControl {
```

Comment:
and Pauseable

Reply:
Asset Create will likely be the only user of this vault and it is already pausable. Not sure if we need to pause it because we don't allow user interactions with the vault anyway.